tests/test_routes.py

```diff
@@ -0,0 +1,203 @@
+import pytest
+from app.models.board import Board
+from app.models.card import Card
+
+#Board tests:
+
+def test_get_one_saved_board_no_cards(client, one_board_no_cards):
+    #Act
+    response = client.get("/boards/1")
+    response_body = response.get_json()
+
+    #Assert
+    assert response.status_code == 200
+    assert "board" in response_body
+
+def test_get_board_not_found(client):
+    #Act
+    response = client.get("/boards/1")
+    response_body = response.get_json()
+
+    #Assert
+    assert response.status_code == 404
+    assert response_body == {"details": "Board id: 1 not found"}
+
+def test_get_boards_no_boards(client):
+    #Act
+    response = client.get("/boards")
+    response_body = response.get_json()
+
+    #Assert
+    assert response.status_code == 200
+    assert response_body == []
+
+def test_get_one_board_with_cards(client, one_board_no_cards):
```

Comment:
I think it's important to be really clear about what we are testing with our naming. This tests takes in a board with no cards, adds 2 cards, then fetches it to check the contents. To me, the adding cards portion is important to note, since we are executing additional code than if we had a test which passed in a fixture of a board that already had 2 cards attached to it and immediately fetched that board.